string robot_name
int64 grassPX
int64 grassPY
